x-pack/filebeat/input/httpjson/input.go

```diff
@@ -161,8 +161,8 @@ func run(
 	return nil
 }
 
-// The Request.Tracer.Filename may have ":" when a httpjson input has cursor config
-// The MacOs Finder will treat this as path-separator and causes to show up strange filepaths.
+// The request.tracer.filename may have ":" when a httpjson input has cursor config
```

Comment:
The typical [godoc](https://go.dev/blog/godoc) convention is to start the first sentence with the name of the element being documented.

`// sanitizeFileName replaces ":" and "/" with "_". ...`